tests/ParserTest.php

```diff
@@ -193,37 +194,37 @@ public function unicodeParsing(): void
             }
             $contentRules = $ruleSet->getRules('content');
             $firstContentRuleAsString = $contentRules[0]->getValue()->__toString();
-            if ($selector == '.test-1') {
+            if ($selector === '.test-1') {
```

Comment:
It won't make any difference comparing with strings other than "0", but better to use `===` anyway.